docs/theory/apportion_abundance.md

```diff
@@ -84,21 +103,23 @@ $$
 \end{equation}
 $$
 
-These are then summed across all fish of $s$:
+Therefore, the total numbers of male and female aged fish of length $\ell$ and age $\alpha$ are:
 
 $$
 \begin{equation}
 \begin{aligned}
-    n_{\textrm{M}}^{\textrm{aged}} &= \sum_{\textrm{M},\ell,\alpha}n_{\textrm{M},\ell,\alpha}^{\textrm{aged}} \nonumber \\
-    n_{\textrm{F}}^{\textrm{aged}} &= \sum_{\textrm{F},\ell,\alpha}n_{\textrm{F},\ell,\alpha}^{\textrm{aged}} \nonumber \\    
+    n_{\textrm{M}}^{\textrm{aged}} &= \sum_{\ell,\alpha}n_{\textrm{M},\ell,\alpha}^{\textrm{aged}} \nonumber \\
+    n_{\textrm{F}}^{\textrm{aged}} &= \sum_{\ell,\alpha}n_{\textrm{F},\ell,\alpha}^{\textrm{aged}} \nonumber \\    
 \end{aligned}
 \label{eq:total_aged_sex} \tag{7}
 \end{equation}
 $$
 
+
+
 ### Number proportions
 
-The sex-specific abundances for unaged $\eqref{eq:total_unaged_sex}$ and aged $\eqref{eq:total_aged_sex}$ fish are then summed together to calculate the total unaged ($n^{\textrm{unaged}}$), aged ($n^{\textrm{aged}}$), and all ($n$) fish:
+The sex-specific numbers for unaged $\eqref{eq:total_unaged_sex}$ and aged $\eqref{eq:total_aged_sex}$ fish are then summed to calculate the total number of unaged fish($n^{\textrm{unaged}}$), aged ($n^{\textrm{aged}}$), and all ($n$) fish:
```

Comment:
Perhaps 'numbers' should be replaced with 'counts'? I suppose it is not a huge deal so long as we are consistent in usage.

Reply:
Sounds good!

Reply:
Right now everywhere else we use "numbers," including in the glossary. I have no problem with changing it to "counts" if it is more clearer.

Reply:
oh wait, wouldn't "number proportion" then need to be "count proportion"? it sounds a little weird.

Reply:
Aye, that's true. I think it's fine to use 'number' so long as we are consistent.